app/controllers/api/v1/users_controller.rb

```diff
@@ -76,8 +76,9 @@ def orders_count
       def mentionable_users
         return render json: { users: [] } if params['roles'].nil?
 
-        @users = User.mentionable_users({ roles: params[:roles],
-                                          order_id: params[:order_id] })
+        @users = User.mentionable_users( roles: params[:roles],
+                                         order_id: params[:order_id],
```

Comment:
Same as the comment below, messages are not order-specific, so if possible this shouldn't either